deploy.sh

```diff
@@ -1,38 +1,45 @@
 #!/bin/sh
 
 # Scripts to deploy application to staging server
-# @todo #45:30min Continue implementing deploy scripts. Find out user 
-#  and domain of the staging server and replace scp and ssh with correct data.
-#  Add scripts in cron (like the one created in #47). Verify the web 
-#  application is running (execute couple fo curl requests).
+# @todo #126:30min Finish implementing deploy scripts. Add scripts in cron 
+#  (like the one created in #47). Verify the web application is running - 
+#  execute couple of curl requests.
 
```

Comment:
@bkuzmic I would also add a puzzle to add another parameter to decide in which environment to deploy, either 'staging' or 'production'

Reply:
@emilianodellacasa Sure, I'll do that.